### ISSUE TITLE
"Sign in" Dialog's disabled "Sign in" button has limited color contrast

### ISSUE THREAD

**Issue by outofambit** [desktop/desktop#6576]
Advisory as this is a disabled button. Ratio of only 2.6:1 does not meet the requirement of 4.5:1

![tdna_gh_desk_win_017](https://user-images.githubusercontent.com/964912/50984769-7a017f80-14b7-11e9-9e76-19bd1deebd61.png)

cc @ampinsk

**Comment by ampinsk** [comment]
If we increase the contrast on disabled buttons, they won't look disabled anymore. [According to W3C](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html), it's fine for disabled buttons to not pass contrast: 

> Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.
